ansible/roles/test/tasks/everflow/config_test/config_test.yml

```diff
@@ -1,173 +1,275 @@
 - name: Define variables for test
   set_fact:
     testname: config_test
-    config_files:
-        - config_src_ip_invalid_1.json
```

Comment:
since these files are no longer used, can you remove them?

Reply:
@lguohan 
Just removed those useless from branch.